pipelines/hello_world_kubeflow_pipelines/README.md

```diff
@@ -0,0 +1,249 @@
+## Getting started with notebook pipelines
+
+A notebook pipeline comprises of notebook nodes that are connected with each other. While pipelines can be used to run any kinds of notebooks, they are commonly used to implement machine learning workflows.
+
+![The completed tutorial pipeline](doc/images/completed_tutorial_pipeline.png)
+
+Notebook pipelines can run locally in JupyterLab or remotely on Kubeflow Pipelines, a platform for building and deploying machine learning workflows based on Docker containers.
+
+![Run a notebook pipeline locally or on Kubeflow Pipelines](doc/images/notebook_pipeline_local_and_remote.png)
+
+In this tutorial you will learn how to create a notebook pipeline and run it on Kubeflow Pipelines. Take a look at the [Hello World tutorial](https://github.com/elyra-ai/examples/tree/master/pipelines/hello_world) to learn how to run notebook pipelines locally in JupyterLab. 
+
+### Prerequisites
+
+To complete this tutorial you need
+- [JupyterLab 2.x with the Elyra extension v1.1 (or newer) installed](https://elyra.readthedocs.io/en/latest/getting_started/installation.html)
+- Access to a [Kubeflow Pipelines](https://www.kubeflow.org/docs/pipelines/overview/pipelines-overview/) deployment
+
```

Comment:
Do you want to mention that, if they don't have access to KFP, they could deploy one locally if they have the resources? https://elyra.readthedocs.io/en/latest/recipes/deploying-kubeflow-locally-for-dev.html

Reply:
Yep, will do.

Reply:
done